.circleci/config.yml

```diff
@@ -121,6 +117,9 @@ workflows:
           filters:
             tags:
               ignore: /^v.*/
+      - cypress:   
+          requires:
+            - build 
```

Comment:
Does it make sense to run this directly after the build? Is the new image already deployed when Cypress starts? Otherwise we should probably just keep the nightly job.